core/settings.py

```
@@ -29,7 +29,11 @@
 SECRET_KEY = os.environ.get('SECRET_KEY', 'django-insecure-dev-key-for-local-testing')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
+IS_PRODUCTION = bool(os.environ.get('VERCEL'))
+if IS_PRODUCTION:
+    DEBUG = False
+else:
+    DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
```

Ruff (0.15.14)

[warning] Use ternary operator `DEBUG = False if IS_PRODUCTION else os.environ.get('DEBUG', 'True').lower() == 'true'` instead of `if`-`else`-block

Replace `if`-`else`-block with `DEBUG = False if IS_PRODUCTION else os.environ.get('DEBUG', 'True').lower() == 'true'`

(SIM108)